src/AcronymList.js

```diff
@@ -35,7 +36,19 @@ const isLiked = (likesIds, definitionId) => {
 };
 
 const AcronymList = (props) => {
-  const { list, like, isAuthenticated, likesIds } = props;
+  const {
+    list = [],
+    like,
+    isAuthenticated,
+    likesIds
+  } = props;
+
+  if (!list.length || !list.map) {
```

Comment:
can a list not have map but have a length?

Reply:
If list is a map, list.length will be undefined, so this if statement will evaluate to true and return <NoResultsPage /> regardless of the list value.

Reply:
Added an update that should handle that.